CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# Adding `mobile-ui-testing-squad` as codeowners for UI tests related folders
+
```

Comment:
My two cents: I think we can do without this comment. The syntax is straightforward enough.

Reply:
Good point, I removed the comment in e93bb335fd66df34aa5db354014d924d1088145c.